.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: Verify
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ 8, 11, 17 ]
```

Comment:
21 seems to work, I suggest replacing 17 with 21 since it can be usually implied that things work on 17 if they work on 21.

Reply:
For me build on 21 fails with some SecurityManager related problem... will dblcheck

Reply:
no worries, if it fails its fine. I noticed I used an old copy of the repo since I was testing something else while checking.